src/main/java/com/google/sps/data/Project.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.data;
+
+/** Class containing projects*/
+public final class Project {
+
+  private final double id;
+  private String title;
+  private String summary;
+
+  private Project (ProjectBuilder builder) {
+    this.id = builder.id;
+    this.title = builder.title;
+    this.summary = builder.summary;
+  }
+
+  public double getId() {
+    return id;
+  }
+
+  public String getTitle() {
+    return title;
+  }
+
+  public String getSummary() {
+    return summary;
+  }
+
+  /** Class used to build projects*/
+  public static class ProjectBuilder {
+
+    private final double id;
+    private String title;
+    private String summary;
+
+    public ProjectBuilder(double id) {
```

Comment:
[optional] Should a title also be required?

Reply:
I decided to only use an id, because it is the only value that is required to query the projects from the API. So the id is all you need to get the rest of the information.